Makefile

```diff
@@ -103,6 +103,7 @@ gomodtidy: gomods
 .PHONY: mockery
 mockery: $(mockery) ## Install mockery.
 	go install github.com/vektra/mockery/v2@v2.43.2
+	mockery
```

Comment:
This might be surprising. Should at least update the comment to be consistent.

On the other hand, I think it would be expected for this to be included in `generate` down below.

Reply:
both are named the same so it's a bit confusing

Reply:
Done